### PR TITLE
Fix for adding extra OpenId providers (ref #1509)

### LINKED ISSUE #1509: Exposed providers in openid capabilities are not filtered by `multiauth.policies`
For example, with:

```ini

kinto.includes = kinto.plugins.openid

multiauth.policies = google

multiauth.policy.google.use = kinto.plugins.openid.OpenIDConnectPolicy
multiauth.policy.google.issuer_url = https://accounts.google.com
multiauth.policy.google.client_id = a
multiauth.policy.google.client_secret = b

multiauth.policy.fxa.use = kinto.plugins.openid.OpenIDConnectPolicy
multiauth.policy.fxa.issuer_url = https://accounts.mozilla.org
multiauth.policy.fxa.client_id = a
multiauth.policy.fxa.client_secret = b
```

`fxa` provider should not be shown in capability.

See misbehaving code (OMG a `print()` statement!)

https://github.com/Kinto/kinto/blob/bf60689ae016a31b8c5bb1955397be46d7b4acca/kinto/plugins/openid/__init__.py#L89-L95

### PR DESCRIPTION
**Pull request by schobewankenobi**
Filters OpenId policies in settings on multiauth.policies

Fixes #1509

### CODE REVIEW

File: kinto/plugins/openid/__init__.py [hunk 1]
```diff
@@ -87,11 +87,10 @@ def includeme(config):
     settings = config.get_settings()
 
     openid_policies = []
-    for k, v in settings.items():
-        m = re.match('multiauth\.policy\.(.*)\.use', k)
-        if m:
-            if v.endswith('OpenIDConnectPolicy'):
-                openid_policies.append(m.group(1))
+    for policy in settings['multiauth.policies'].split(" "):
```
Comment (leplatrem) [inline kinto/plugins/openid/__init__.py]: it is recommended to use `aslist()` from `pyramid.settings`

**Comment by schobewankenobi** [comment]
Requesting another review @leplatrem , since Trvis is finally building correctly.

**Comment by jeff-tista** [comment]
Added a line, let me know if that's satisfactory. Thanks!

**Comment by jeff-tista** [comment]
Travis seems to be failing for an unrelated reason..hm.

**Comment by glasserc** [comment]
I can help with the travis failures. It looks like it just failed to build docs due to a network error. I restarted the build. I'll keep an eye on it for a little while and let you know if I think any action is required on your part.

**Comment by glasserc** [comment]
Thanks for your contribution!

**Comment by jeff-tista** [comment]
Thanks for the opportunity!

**Pull request merged**